linear_feedback_controller/include/linear_feedback_controller/contact_detector.hpp

```diff
@@ -0,0 +1,93 @@
+#ifndef LINEAR_FEEDBACK_CONTROLLER_CONTACT_DETECTOR_HPP
+#define LINEAR_FEEDBACK_CONTROLLER_CONTACT_DETECTOR_HPP
+
+#include <Eigen/Core>
+
+namespace linear_feedback_controller {
+
+class ContactDetector {
+ public:
+  /**
+   * @brief State of the contact:
+   *  - NO_CONTACT: There is no contact.
+   *  - GOING_TO_CONTACT: The contact is settling down but not active yet.
+   *  - ACTIVE_CONTACT: There is contact.
+   *  - RELEASING_CONTACT: The contact is starting to get released.
+   */
+  enum ContactState {
+    NO_CONTACT = 0,
```

Comment:
why NO_CONTACT=0?

Reply:
this initialize the enum first item
if you say NO_CONTACT = 123 the next one is always `previous + 1`